File: CAPITULO 02/Java/Ex03.java
import java.util.*;
import java.io.*;
public class Ex03{
	public static void
		 main(String[]args){
		int n1,n2;
		Scanner num = new Scanner (System.in);
		System.out.println("Digite um numero1");
		n1 = num.nextInt();
		System.out.println("Digite o numero2");
		n2 = num.nextInt();
		if (n1>n2)
			System.out.println("Numero1 maior");
		else
			System.out.println("Numero2 maior");
			
	}
}

File: CAPITULO 02/C++/ex03.c
#include <stdio.h>
int main (){
    int n1,n2,maior;
    printf("Digite um numero1 :");
    scanf("%f%*c",&n1);
    printf("Digite um numero2 :");
    scanf("%f%*c",&n2);
    if(n1>n2)
        printf("numero1 maior que o numero2\n");
    else
        printf("numero2 maior que o numero1\n");

    system("pause");
    return 0;
}
